### DISCUSSION TITLE
Custom quest helper ideas

### DISCUSSION THREAD

**Discussion by pajlada** [Zoinkwiz/quest-helper#1871] [Ideas]
Zoinkwiz is looking into making it possible for users to create & share custom helpers.

What are some detailed examples of what you'd want to be able to achieve with this (e.g. league start helpers, new account helpers)

Keep in mind that not everything available to a custom made quest helper would be available here, and these examples you provide would be used to figure out what to make available to these custom helpers.

As a bad example:
1. Thieve men until 40 thieving (Skill Requirement, Highlight Men with a world point)
2. Steal fruit until 60 thieving (Skill Requirement, Highlight Object with a world point)
3. Finish curse of arrav ( Just a varbit requirement)
4. run agility laps in draynor until 20 agiity (Skill Requirement, Just an arrow at the agility course start)
5. Kill Ogres until you loot a Dragon 2h Dragon Mace, (ItemRequirement, Highlight Ogres with a world point)

**Comment by lexi-sh** [comment]
Not sure if this has been suggested, or if this is a good example, but what about combat achievements? A helper for each of the tiers of combat achievements, where it would help direct the player to the area of each enemy required, as well as show the requirements, if any, to get the achievement (other than killing)
